wallet-new/src/Cardano/Wallet/Client.hs

```diff
@@ -58,6 +58,8 @@ data WalletClient m
     , getAddressValidity
          :: Text -> Resp m AddressValidity
     -- wallets endpoints
+    , importWalletKey
+         :: FilePath -> Resp m Wallet
```

Comment:
let's remove this and leave that to a further ticket for implementing/testing this endpoint specifically